root/programs/C#/Frameworks/Infrastructure/Public/IO/SymmetricCryptography.cs

```diff
@@ -438,9 +438,9 @@ private static SymmetricAlgorithm CreateSymmetricAlgorithm(EnumSymmetricAlgorith
         {
             SymmetricAlgorithm sa = null;
 
-            // AesCryptoServiceProvider, AesManagedは3.5からの提供。
-            // 暗号化プロバイダ選択のポリシー（Managed → CSP → CNG）
-            // に合わせて、AesCryptoServiceProviderを削除。
+            // AesCryptoServiceProvider, AesManagedは.NET Framework 3.5からの提供。
+            // 暗号化プロバイダ選択の優先順は、Managed → CAPI(CSP) → CNG。
```

Comment:
@daisukenishino 

If this order is *descending order*, is the priority of CAPI higher than CNG?
After Windows Vista, CAPI was replaced with CNG.
So, I suspect that the priority of CNG is higher than CAPI.